client/me/help/help-contact/index.jsx

```diff
@@ -211,16 +210,14 @@ module.exports = React.createClass( {
 	},
 
 	onOperatorsAway: function() {
-		const { isOlarkReady, isUserEligible, details } = this.state.olark;
-		const showChatVariation = isUserEligible && details.isOperatorAvailable;
-		const showKayakoVariation = ! showChatVariation && ( details.isConversing || isUserEligible );
-		const showForumsVariation = ! ( showChatVariation || showKayakoVariation );
+		const { details } = this.state.olark;
 
 		if ( ! details.isConversing ) {
 			analytics.tracks.recordEvent( 'calypso_help_offline_form_display', {
-				form_type: showKayakoVariation ? 'kayako' : 'forum'
+				form_type: 'kayako'
```

Comment:
I felt I should show my work here just so someone else could try and spot a mistake. Here's how I reduced it

```js
showKayakoVariation = ! showChatVariation && ( details.isConversing || isUserEligible );
showKayakoVariation = ! ( isUserEligible && details.isOperatorAvailable ) && ( details.isConversing || isUserEligible ); // substitute showChatVariation for its actual value
showKayakoVariation = ! ( isUserEligible && false ) && ( details.isConversing || isUserEligible ); // We already know operators aren't available so lets replace details.isOperatorAvailable with false because we are in the onOperatorsAway callback
showKayakoVariation = ! ( false ) && ( details.isConversing || isUserEligible ); // simplify
showKayakoVariation = true && ( details.isConversing || isUserEligible ); // simplify
showKayakoVariation = ( details.isConversing || isUserEligible ); // simplify
showKayakoVariation = details.isConversing || isUserEligible; //simplify
showKayakoVariation = false || isUserEligible; // Our if statement above indicates that details.isConversing should be false so lets fill it in
showKayakoVariation = isUserEligible; // simplify
showKayakoVariation = false; // If a user is not eligible to start a chat and they aren't currently chatting then they'd
                                                // never experience an "offline" contact form. In this case isUserEligible will always be false.
```